internal/database/database.go

```diff
@@ -194,7 +194,8 @@ CREATE TABLE reverseSwaps
     paidAt              INT,
     externalPay         BOOLEAN,
     walletId            INT REFERENCES wallets (id) ON DELETE SET NULL,
-    tenantId            INT REFERENCES tenants (id)
+    tenantId            INT REFERENCES tenants (id),
+    maxRoutingFeePpm    INT DEFAULT 0
```

Comment:
When you default to 0 how do you differentiate between an actual 0 ppm limit and when to use the configured default?